addons/crm_livechat/__manifest__.py

```diff
@@ -23,7 +23,11 @@
             'crm_livechat/static/src/core/*',
         },
         'web.assets_unit_tests': [
-            'crm_livechat/static/tests/**/*',
+            'crm_livechat/static/tests/**.test.js',
         ],
+        'web.qunit_suite_tests': [
```

Comment:
Note: I had to update the manifest because the dependencies were missing. Is it okay to do that?

Reply:
New tests written in 17.2 should be rewritten with HOOT directly.